plugins/available/rbenv.plugin.bash

```diff
@@ -3,9 +3,7 @@
 cite about-plugin
 about-plugin 'load rbenv, if you are using it'
 
-pathmunge "${HOME}/.rbenv/bin"
+pathmunge $HOME/.rbenv/bin
```

Comment:
Sorry, that was a mistake. It fails to work if `$HOME` contains space. Should I make another commit with corrections?